src/Integration/LocalServices/IProjectSystemHelper.cs

```diff
@@ -132,5 +132,12 @@ internal interface IProjectSystemHelper : ILocalService
         /// Returns a flag indicating whether there is fully-opened solution or not.
         /// </summary>
         bool IsSolutionFullyOpened();
+
+        /// <summary>
+        /// Returns true if the project is implemented by the old C#/VB project system, otherwise false
```

Comment:
nitpick: would be useful to say what 'old' means - 2 years from now it may mean something else